libraries/Microsoft.Bot.Builder/Adapters/TestFlow.cs

```diff
@@ -280,14 +280,14 @@ public TestFlow AssertReply(IActivity expected, IEqualityComparer<IActivity> equ
                     description = description ?? expected.AsMessageActivity()?.Text.Trim();
                     if (expected.Type != reply.Type)
                     {
-                        throw new Exception($"{description}: Type should match");
+                        throw new ArgumentException($"{description}: Type should match");
                     }
 
                     if (equalityComparer != null)
                     {
                         if (!equalityComparer.Equals(expected, reply))
                         {
-                            throw new Exception($"Expected:{expected}\nReceived:{reply}");
+                            throw new ArgumentException($"Expected:{expected}\nReceived:{reply}");
```

Comment:
I'm not sure these should be ArgumentExceptions. Maybe `InvalidOperationException` suits better for assertions.

Reply:
I think `ArgumentException` is more specific, but both work fine.

Reply:
fixed, thanks